unity-renderer/Assets/ABConverter/Core.cs

```diff
@@ -156,6 +157,14 @@ void UpdateLoop()
                             
                             if (value.IsCompleted)
                             {
+                                FileInfo gltfFilePath = new FileInfo(key);
+                                var metrics = new Dictionary<string, object>()
+                                {
+                                    { "meshes_size", value.meshesEstimatedSize },
+                                    { "animation_size", value.animationsEstimatedSize }
+                                };
+                                File.WriteAllText($"{gltfFilePath.Directory.FullName}/metrics.json", JsonConvert.SerializeObject(metrics, Formatting.Indented));
```

Comment:
Do we need this to be a Dictionary?

Wouldn't this better be a `new AssetBundleMetrics  { meshes_size = value.meshesEstimatedSize, animation_size = value.animationsEstimatedSize };` and then just serialize it with `JsonUtility.ToJson()` as we do for the metadata here?
https://github.com/decentraland/unity-renderer/blob/701f1c3b9aada74b68bfe958160bb697c7572541/unity-renderer/Assets/ABConverter/AssetBundleMetadataBuilder.cs#L26

Reply:
Oh my... I forgot to port that code, nice catch!